src/HeroIcon/HeroIcon.js

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { dcnb } from 'cnbuilder';
+import { VideoCameraIcon, MicrophoneIcon } from '@heroicons/react/outline';
+import {
+  ArrowRightIcon,
+  ChevronDownIcon,
+  ChevronRightIcon,
+  DownloadIcon,
+  MailIcon,
+  PlayIcon,
+} from '@heroicons/react/solid';
+import { SrOnlyText } from '../SrOnlyText/SrOnlyText';
+import { heroIconOptions } from './HeroIcon.levers';
+
+export const HeroIcon = ({ icon, srText, isAnimate, className, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  let baseStyle = 'su-w-08em su-ml-03em su--mt-02em';
+  let animateStyle = 'group-hocus:su-translate-x-02em';
+  let transformStyle = '';
+
+  // Levers
+  // ---------------------------------------------------------------------------
+  if (isAnimate) {
+    transformStyle = 'su-transform-gpu';
+  }
+
+  let defaultIcon = (
+    <ArrowRightIcon
+      aria-hidden="true"
+      className={dcnb(
+        'su-transition',
+        baseStyle,
+        animateStyle,
+        transformStyle,
+        className
+      )}
+      {...props}
+    />
+  );
+
+  if (icon && heroIconOptions.includes(icon)) {
+    switch (icon) {
+      case 'video':
+        baseStyle = 'su-w-08em su-mt-[-0.2em]';
+        animateStyle = 'group-hocus:su-translate-x-02em';
+        defaultIcon = (
+          <VideoCameraIcon
+            aria-hidden="true"
+            className={dcnb(
+              'su-transition',
+              baseStyle,
+              animateStyle,
+              transformStyle,
+              className
+            )}
+            {...props}
+          />
+        );
+        break;
+
+      case 'play':
+        baseStyle = 'su-w-08em su-ml-7 su--mt-3';
+        animateStyle = 'group-hocus:su-translate-x-02em';
+        defaultIcon = (
+          <PlayIcon
+            aria-hidden="true"
+            className={dcnb(
+              'su-transition',
+              baseStyle,
+              animateStyle,
+              transformStyle,
+              className
+            )}
+            {...props}
+          />
+        );
+        break;
```

Comment:
You could probably reduce the amount of repeated code if you define a variable to swap out the icon name, e.g., take a look at the refactored one in SAA 😄  I defined a const Icon, and just assigned the HeroIcon name to it depending on the case
https://github.com/SU-SWS/saa_alumni/blob/main/src/components/simple/heroIcon.js

Reply:
@rebeccahongsf The animation handling can be totally different, since here have an extra "animate" option in Decanter React - so just take what you need and discard the rest 😄

Reply:
I had that in a previous commit but for some reason it doesn't seem to re-render the generated SVG when switching the icon type in the Storybook interface.

I  cherry-picked the [PR](https://github.com/SU-SWS/decanter-react/pull/73/files#diff-407bcae5e2086a8aeb39f032c6b834579d18d10ecca33b3f40440d28d8076b8b) onto another branch so that we can see the issue I encountered in the [preview build](https://deploy-preview-73--decanter-react.netlify.app/?path=/story/simple-hero-icon--default) — perhaps my variation of the SAA one isn't structured correctly? 🤔

Reply:
Ah — I think I have an idea of how to make it work! I'll put it together now but let me know if you happen to stumble upon anything else funky or have any additional feedback! 😃

Reply:
Interesting - thanks for testing 😄

Reply:
Interesting x2! I tweaked the original switch case I had modified and now it works 😬 
https://deploy-preview-71--decanter-react.netlify.app/?path=/story/simple-hero-icon--video